ciphers/mono_alphabetic_ciphers.py

```diff
@@ -0,0 +1,65 @@
+LETTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+
+def translate_message(key, message, mode):
+    """
+    >>> translate_message("QWERTYUIOPASDFGHJKLZXCVBNM", "When you do the common things in life in an uncommon way, you will command the attention of the world", "encrypt")
```

Comment:
```suggestion
    >>> translate_message(
    ...     "QWERTYUIOPASDFGHJKLZXCVBNM", 
    ...     "When you do the common things in life in an uncommon way, "
    ...     "you will command the attention of the world", "encrypt")
```
The automated tests hate lines longer than 88 characters.